### PR TITLE
drop down menu added pull request for issue #45

### PR DESCRIPTION
**Pull request by nazrul7711**
hey @anicmarko9  i have added the features this is how it now looks like 
Screenshot 2023-07-04 at 11 23 02 AM
Screenshot 2023-07-04 at 11 28 32 AM

### CODE REVIEW

**Review by anicmarko9 — CHANGES_REQUESTED** [review]
I've updated your changes to guide you in the right direction. See them and implement in your code, and make the PR.

- With these changes, the subLinks are breaking text into two rows. You need to apply CSS to either make font smaller, or make container absolute and stretchable.
- With these changes, CSS in hamburger menu needs adjustments, I'm specifically talking about this line of updated code in HeaderLinkGroup.tsx:
```tsx
                 {subLinks.map(
                      ({ subLink, subTitle }, index: number): JSX.Element => (
                        <li key={`${subLink}-${index}`}>
                          <Link
                            className={`hoverEffect ${
                              isHamburger ? "hover:text-secondary" : "p-3"
                            }`}
                            href={subLink}
                          >
                            {subTitle}
                          </Link>
                        </li>
                      )
                    )}
```

File: src/components/links/HeaderLinkGroup.tsx
[the remark this answers is not in the document]
Comment (nazrul7711) [reply src/components/links/HeaderLinkGroup.tsx:1]: updated

**Comment by nazrul7711** [comment]
i have modified the css and after making changes that u suggested the text is not broken this is the preview don't worry about the background of drop down menu I will change it.
plz also guide me about the doubt I have about hamburger menu I made a comment about that .
Screenshot 2023-07-05 at 10 04 08 AM

**Comment by anicmarko9** [comment]
You can design however you think it's the easiest way. But my point was that I don't want to **hover** over Hamburger Menu Icon and see Nav links like that. There is already clickable functionality, so when you **click** on the menu, it will display container with Nav links all over the screen. I wanted to make **hover** effects on those links that show up, not on a **Hamburger Icon**.

I don't have anything specific in mind about those hover effects, so do whatever you like.

**Comment by nazrul7711** [comment]
ok I will do the needful.

**Review by anicmarko9 — APPROVED** [review]
I'll approve this without testing.

Why are sublinks empty? In Header.tsx you should put each sublink like this:
```diff
+ subLinks: [
+   { subLink: "/faq", subTitle: "FAQs" },
+   { subLink: "/support", subTitle: "Support Center" },
+   { subLink: "/contact", subTitle: "Contact Us" },
+ ]
```

Sublinks titles are similar to ones in Footer.tsx

**Pull request merged**